applications/FluidDynamicsApplication/custom_elements/two_fluid_navier_stokes.cpp

```diff
@@ -1942,6 +1942,47 @@ void TwoFluidNavierStokes<TElementData>::load(Serializer &rSerializer)
     KRATOS_SERIALIZE_LOAD_BASE_CLASS(rSerializer, BaseType);
 }
 
+
+template <class TElementData>
+void TwoFluidNavierStokes<TElementData>::GetValueOnIntegrationPoints(   const Variable<double> &rVariable,
+                                                                        std::vector<double> &rValues,
+                                                                        const ProcessInfo &rCurrentProcessInfo )
+{
+    if (rVariable == DIVERGENCE){
+
+        KRATOS_CHECK_VARIABLE_KEY( DIVERGENCE );
+
+        GeometryType::Pointer p_geom = this->pGetGeometry();
+        const GeometryType::IntegrationPointsArrayType& IntegrationPoints = p_geom->IntegrationPoints(GeometryData::GI_GAUSS_2);
+        const unsigned int NumGauss = IntegrationPoints.size();
+
+        if (rValues.size() != NumGauss){
+            rValues.resize(NumGauss);
+        }
+
+        Vector GaussPtsJDet = ZeroVector(NumGauss);
+        GeometryData::ShapeFunctionsGradientsType DN_DX;
+        p_geom->ShapeFunctionsIntegrationPointsGradients(DN_DX, GaussPtsJDet, GeometryData::GI_GAUSS_2);
+
+        for (unsigned int i_gauss = 0; i_gauss < NumGauss; i_gauss++){
+
+            const Matrix gp_DN_DX = DN_DX[i_gauss];
+            double DVi_DXi = 0.0;
+
+            for(unsigned int nnode = 0; nnode < NumNodes; nnode++){
+
+                const Vector vel = (*p_geom)[nnode].GetSolutionStepValue( VELOCITY );
+                for(unsigned int ndim = 0; ndim < Dim; ndim++){
+                    DVi_DXi += gp_DN_DX(nnode, ndim) * vel[ndim];
+                }
+            }
+            rValues[i_gauss] = DVi_DXi;
+        }
+    } else {
+        KRATOS_DEBUG_ERROR << "Variable could not be retrieved from NavierStokesTwoFluid element" << std::endl;
```

Comment:
Please remove this, we ususally do nothing if output is not implemented for a certain variable

Reply:
Yes. Done.